docs/guide/basic-module.md

```diff
@@ -137,20 +116,25 @@ If we re-run stencil and look at `hello.go` we should see the following:
 package main
 
 func main() {
-	fmt.Println("Hello, world!")
-	// <<Stencil::Block(additionalMessage)>>
+  fmt.Println("Hello, world!")
+  // <<Stencil::Block(additionalMessage)>>
 
-	// <</Stencil::Block>>
+  // <</Stencil::Block>>
 }
 ```
 
-If we add contents to the block and re-run stencil they'll be persisted across the run!
+If we add contents to the block and re-run stencil they'll be persisted
+across the run!
 
 ## Step 5: (Optional/Advanced) Creating Multiple Files
 
-One of the powerful parts of stencil is the ability to create an arbitrary number of files with a single template. This is done with the [`file.Create`](/functions/file.Create) function. Let's create a `greeter.go.tpl` template in the `templates/` directory that'll create `<greeting>.go` based on the `greetings` argument.
+One of the powerful parts of stencil is the ability to create an
+arbitrary number of files with a single template. This is done with the
+[`file.Create`](/functions/file.Create) function. Let's create a
+`greeter.go.tpl` template in the `templates/` directory that'll create
+`<greeting>.go` based on the `greetings` argument.
 
-```tpl
+```yaml
```

Comment:
p sure this is not yaml

Reply:
yeah but `tpl` wasn't exactly doing anything so i picked something that looked okay enough lmao

Reply:
remove the `'` made it look better w/ Go so switched to that